public/app/features/browse-dashboards/components/CreateNewButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { Button, ButtonGroup, Dropdown, Menu, MenuItem } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+interface Props {
+  /**
+   * Pass a folder UID in which the dashboard or folder will be created
+   */
+  inFolder?: string;
+}
+
+export function CreateNewButton({ inFolder }: Props) {
+  const newMenu = (
+    <Menu>
+      <MenuItem
+        url={addFolderUidToUrl('/dashboard/new', inFolder)}
+        label={t('search.dashboard-actions.new-dashboard', 'New Dashboard')}
+      />
+      <MenuItem
+        url={addFolderUidToUrl('/dashboards/folder/new', inFolder)}
+        label={t('search.dashboard-actions.new-folder', 'New Folder')}
+      />
+      <MenuItem
+        url={addFolderUidToUrl('/dashboard/import', inFolder)}
+        label={t('search.dashboard-actions.import', 'Import')}
+      />
+    </Menu>
+  );
+
+  return (
+    <Dropdown overlay={newMenu}>
+      <ButtonGroup>
+        <Button icon="plus">New</Button>
+        <Button icon="angle-down" />
+      </ButtonGroup>
```

Comment:
Hmm. I know Figma has the split buttons, but it's a bit wonky to have both the main "New" section and the split chevron do the same thing of open the dropdown menu.

![image](https://user-images.githubusercontent.com/46142/234543939-1534613e-027a-4682-9207-4e183c49f98e.png)


@amy-super Is there a primary action that clicking "New" should trigger, different to dropdown?

Reply:
@tskarhed as we discussed today, we can change this to a regular button like old UI


```tsx
<Dropdown overlay={MenuActions} placement="bottom-start">
  <Button variant="primary">
    New
    <Icon name="angle-down" />
  </Button>
</Dropdown>
```